### PR TITLE
Added support to Color feature

### PR DESCRIPTION
**Pull request by iturcios**
Added support to Color feature

### CODE REVIEW

File: src/main/java/faker/generator/FakeColor.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package faker.generator;
+
+import java.lang.annotation.*;
+
+@Documented
+@Retention(value = RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+@Inherited
+public @interface FakeColor {
+    FakeColorType value() default FakeColorType.NAME;
```
Comment (mauriciojovel) [inline src/main/java/faker/generator/FakeColor.java]: Remove the unnecessary value, you can do this only with the FakeColor

File: src/main/java/faker/generator/FakeColorType.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package faker.generator;
```
Comment (mauriciojovel) [inline src/main/java/faker/generator/FakeColorType.java]: Remove this class

File: src/main/java/faker/generator/FakeColor.java (hunk 1 above)
Comment (iturcios) [reply src/main/java/faker/generator/FakeColor.java]: Done.

File: src/main/java/faker/generator/FakeColorType.java (hunk 2 above)
Comment (iturcios) [reply src/main/java/faker/generator/FakeColorType.java]: Done

**Pull request merged**